README.md

```diff
@@ -539,30 +524,271 @@ services.Configure<TargetingEvaluationOptions>(options =>
+
+``` JavaScript
+"ShoppingCart": {
+    "Assigner": "Targeting",
+    "Variants": [
+        {
+            "Default": true,
+            "Name": "Big",
+            "ConfigurationReference": "ShoppingCart:Big",
+            "AssignmentParameters": {
+                "Audience": {
+                    "Users": [
+                        "Alec",
+                    ],
+                    "Groups": [
+                        {
+                            "Name": "Ring0",
+                            "RolloutPercentage": 100
+                        },
+                        {
+                            "Name": "Ring1",
+                            "RolloutPercentage": 50
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "Name": "Small",
+            "ConfigurationReference": "ShoppingCart:Small",
+            "AssignmentParameters": {
+                "Audience": {
+                    "Users": [
+                        "Susan",
+                    ],
+                    "Groups": [
+                        {
+                            "Name": "Ring1",
+                            "RolloutPercentage": 50
+                        }
+                    ],
+                    "DefaultRolloutPercentage": 80
+                }
+            }
+        }
+    ]
+}
+```
+
+Based on the configured audiences for the variants included in this feature, if the application is executing under the context of a user named `Alec` then the value of the `Big` variant will be returned. If the application is executing under the context of a user named `Susan` then the value of the `Small` variant will be returned. If a user match does not occur, then group matches are evaluated. If the application is executing under the context of a user in the group `Ring0` then the `Big` variant will be returned. If the user's group is `Ring1` instead, then the user has  a 50% chance between being assigned to `Big` or `Small`. If there is not user match nor group match then the default rollout percentage is used. In this case, 80% of unmatched users will get the `Small` variant leaving the other 20% to get the `Big` variant since it is marked as the `Default`.
+
+Example usage of this assigner can be found in the [FeatureFlagDemo example](./examples/FeatureFlagDemo/Startup.cs#L63).
+
+When using the targeting feature variant assigner, make sure to register it as well as an implementation of [ITargetingContextAccessor](./README.md#ITargetingContextAccessor).
+
+``` C#
+services.AddSingleton<ITargetingContextAccessor, HttpContextTargetingContextAccessor>();
+
+services.AddFeatureManagement();
+        .AddFeatureVariantAssigner<TargetingFeatureVariantAssigner>();
+```
+
+### Variant Value Resolution

```

Comment:
How about just "Variant Resolution"?

The word "value" in this document is confusing to me. We don't assign a "value" to a feature. A variant is simply a configuration of a feature. The configuration can be bound to a strongly typed object, but it doesn't change the fact that it's just the configuration of a feature, not the feature itself.

Reply:
I see. I'd like to discuss this one a bit to see if we can land in the best place with this doc section and the interface associated with it.